### ISSUE TITLE
PLG Production Migration

### ISSUE THREAD

**Issue by andrew-jameson** [raft-tech/TANF-app#3222]
**Description:**
As a follow on to #3046 we want to move the PLG stack from the dev environment to the production environment.

**Acceptance Criteria:** 
_Create a list of functional outcomes that must be achieved to complete this issue_

- [ ] PLG is deployed in prod
- [ ] Dev and Staging apps are still observable from PLG in prod
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
_Create a list of granular, specific work items that must be completed to deliver the desired outcomes of this issue_ 

- [x] Update networking commands in `plg/deploy.sh` to be based on prod instead of dev (make Grafana route private, change `-s` switch to point to prod, etc...)
- [x] Update nginx cloud conf to proxy Grafana in the same way we do locally
- [x] Update `deploy-frontend.sh` networking to correctly proxy Grafana in the same way we do with Kibana
- [x] Use the `plg/deploy.sh` script to deploy PLG AND a prod postgres-exporter to the Prod env
- [x] Use the `plg/deploy.sh` script to re-deploy the postgres-exporter for `dev` and `staging` environments
- [x] Update `deploy-backend.sh` and `deploy-fronend.sh` PLG networking to tunnel routes to prod PLG
- [x] Run Testing Checklist and confirm all tests pass

**Notes:**
_Add additional useful information, such as related issues and functionality that isn't covered by this specific issue, and other considerations that will be helpful for anyone reading this_ 

- _Note 1_
- _Note 2_
- _Note 3_

**Supporting Documentation:**
_Please include any relevant log snippets/files/screen shots_

- _Doc 1_
- _Doc 2_

**Open Questions:**
_Please include any questions or decisions that must be made before beginning work or to confidently call this issue complete_ 

- _Open Question 1_
- _Open Question 2_

**Comment by lhuxraft** [comment]
11/13/2024: Made code changes, made smoke test in dev deployment. Need to confirm with Alex before changes are made

**Comment by lhuxraft** [comment]
11/15: code changes made, merged into prod

**Comment by lhuxraft** [comment]
11/18: Need one more review

**Comment by lhuxraft** [comment]
11/20: On pause right now. Eric is working through issues in QASP environment (waiting on pipelines to rerun)

**Comment by lhuxraft** [comment]
11/25: Currently testing

**Comment by lhuxraft** [comment]
11/26: Discussed RBAC w/ Eric on Grafana, open concerns about dev visibility into production logs. Eric implementing access controls to resolve concerns
